content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -48,8 +48,7 @@ replaces the functionality provided by the `istio-init` container.
     Refer to [Hosted Kubernetes settings](#hosted-kubernetes-settings) for any non-default settings required.
 
 1.  Install Istio CNI and Istio using `istioctl`.
-    Refer to the [Istio install](/docs/setup/install/kubernetes/) instructions and pass `--set cni.enabled=true`
-    and `--set cni.components.cni.enabled=true` options.
+    Refer to the [Istio install](/docs/setup/install/kubernetes/) instructions and pass `--set cni.enabled=true` option.
```

Comment:
Shouldn't this point to the more general installation instructions, instead of quick start?
```suggestion
    Refer to the [Istio install](/docs/setup/install/istioctl/) instructions and pass `--set cni.enabled=true` option.
```